heron/cli/src/python/.heronrc.sample

```diff
@@ -0,0 +1,6 @@
+activate:* --config-path cmd-activate-*                   #command specific
```

Comment:
It will be great if you could rename this file to be heronrc.sample or SAMPLE .heronrc.sample will be hidden in the code and people might not see it.